src/main/java/io/github/homchom/recode/mod/commands/impl/other/BaseVarCommand.java

```diff
@@ -0,0 +1,57 @@
+package io.github.homchom.recode.mod.commands.impl.other;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.context.CommandContext;
+import io.github.homchom.recode.mod.commands.Command;
+import io.github.homchom.recode.mod.commands.arguments.ArgBuilder;
+import io.github.homchom.recode.sys.player.chat.ChatType;
+import io.github.homchom.recode.sys.player.chat.ChatUtil;
+import io.netty.handler.codec.base64.Base64Encoder;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.fabricmc.fabric.api.util.NbtType;
+import net.minecraft.client.Minecraft;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.Tag;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.commands.CommandBuildContext;
+
+import java.util.Base64;
+
+public class BaseVarCommand extends Command {
+    @Override
+    public void register(Minecraft mc, CommandDispatcher<FabricClientCommandSource> cd, CommandBuildContext context) {
+        cd.register(ArgBuilder.literal("basevar").executes(ctx -> {
+            ItemStack heldItem = mc.player.getMainHandItem();
+            CompoundTag tag = heldItem.getTagElement("PublicBukkitValues");
+
+            if (tag != null && tag.getString("hypercube:varitem") != "") {
+                    JsonObject data = JsonParser.parseString(tag.getString("hypercube:varitem")).getAsJsonObject();
+                    if (data.get("id").getAsString().contains("loc") || data.get("id").getAsString().contains("snd") || data.get("id").getAsString().contains("pot") || data.get("id").getAsString().contains("vec") || data.get("id").getAsString().contains("part")) {
```

Comment:
1. Why restrict this, couldn't it work for any code item?
2. You seem to have misunderstood how identifiers work here, use `.equals`.

Reply:
I used .equals, and it didnt work. Also, there are some code items that can just be written in code, and don't really need to be every single code item.